### ISSUE TITLE
Cannot login with LineageOS 14.1

### ISSUE THREAD

**Issue by jpdeplaix** [Vavassor/Tusky#515]
I just installed Tusky under LineageOS 14.1 (with GApps pico) and when I try to log in, I enter my instance name, it redirects me to a web browser window to login, I log in, it works fine in the browser but Tusky is not updated so I retype the instance name again, try to login using my login/password, authorize the Tusky app to manage toots on my account and then it redirects me to yet another web page that tells me the url is not supported (oauth something something).

Why is your application opening web pages to log in whereas some other apps such as [Mastalab](https://github.com/stom79/mastalab) does not and works well for me ?

**Comment by remi6397** [comment]
If you use Gello (the Lineage/Cm default browser), it might be the cause. Could you check with _latest_ Chromium, please?

**Comment by jpdeplaix** [comment]
> If you use Gello (the Lineage/Cm default browser), it might be the cause.

Yes I do. Why is that ?

Where do I install Chromium ? There is no such app in both f-droid and google app store (there is Google Chrome of course, but: no).

**Comment by connyduck** [comment]
Yes, this is a known issue with the Lineage default browser. We plan to have a login like Mastalab one day, but since there is a workaround it does not have any priority.

**Comment by jpdeplaix** [comment]
That's a bit sad :(

**Comment by remi6397** [comment]
> Where do I install Chromium ? There is no such app in both f-droid and google app store (there is Google Chrome of course, but: no).


You can download pre-built binaries [Here](https://download-chromium.appspot.com) , or build it from source if you don't trust that site. Or download a Chromium updater from F-Droid.

**Comment by charlag** [comment]
Firefox works too, BTW.
Authorization ia hard because there's a case with 2FA and it implementing it takes a lot of time. We would be glad if someone implemented it without browser

**Comment by nailyk-fr** [comment]
> Yes, this is a known issue with the Lineage default browser. We plan to have a login like Mastalab one day, but since there is a workaround it does not have any priority.

Sad! Did you tried to contact them? 

@jpdeplaix are you sure your maintainer is using the new browser and not the outdated one (Gello)? The new one should be Chromium IIRC.

**Comment by jpdeplaix** [comment]
@nailyk-fr Probably no. My maintainer's device died a month ago anyway so I cannot get any more easy updates.

**Comment by charlag** [comment]
@kit-ty-kate any updates on this? Does it work in recent versions or it's still broken?

**Comment by kit-ty-kate** [comment]
@charlag you mean recent versions of Tusky ? I just tried and as expected it doesn't work with the default browser (Gello I suppose) but works once Firefox Klar is installed. So I guess the issue still stands ?

**Comment by o-jasper** [comment]
FTR had this issue with DuckDuckGo, but it went away for some reason.
